packages/CodeQuality/src/Rector/Identical/SimplifyIdenticalFalseToBooleanNotRector.php

```diff
@@ -39,21 +39,21 @@ public function refactor(Node $node): ?Node
             $comparedNode = $node->right;
             $shouldUnwrap = $node->right instanceof BooleanNot;
         } else {
-            return $node;
+            return null;
         }
 
         if ($shouldUnwrap) {
             /** @var BooleanNot $comparedNode */
             $comparedNode = $comparedNode->expr;
             if ($this->shouldSkip($comparedNode)) {
-                return $node;
+                return null;
```

Comment:
Why all these related changes? I couldn't find a description or commit message to describe them

Reply:
https://github.com/rectorphp/rector/pull/716/commits/f330c5e9c2f92f11c428425133cacc412ac4358c

To make clear that nothing is changing and Rector stops right there.